Makefile

```diff
@@ -79,5 +82,29 @@ coverage: test
 .PHONY: clean
 clean:
 	$(GO_CLEAN) ./...
-	-rm -rf ./bin
+	-rm -rf bin
 	-rm -rf coverage.txt
+	-rm -rf *.tgz
+	-rm -rf *.tgz
+	-rm -rf *.asc
+	-rm -rf *.sha512
+
+release-src: clean
+	-tar -zcvf $(RELEASE_SRC).tgz \
+	--exclude bin \
+	--exclude .git \
+	--exclude .idea \
+	--exclude .DS_Store \
+	--exclude .github \
+	--exclude $(RELEASE_SRC).tgz \
```

Comment:
Do we need any tips about this is only working in MacOS? Because AFAIK, `--exclude` is only working on MacOS. :)

Reply:
> Do we need any tips about this is only working in MacOS? Because AFAIK, `--exclude` is only working on MacOS. :)

I run `make release` in a Linux docker container, it works fine

Reply:
OK. Last time, I tried this in Centos, it fails :P.